Library/Homebrew/formulary.rb

```diff
@@ -118,6 +121,7 @@ def initialize(alias_path)
       path = alias_path.resolved_path
       name = path.basename(".rb").to_s
       super name, path
+      @install_name = alias_path.basename.to_s
```

Comment:
`unresolved_name` would imply to me that it could be a URL, for a file path, which currently it'll never be…